downloadPdfCustomSelectionToolbar/downloadPdfCustomSelectionToolbar.js

```diff
@@ -46,11 +46,17 @@ var updateDownloadPdfCustomSelectionToolbarSelectButtonLabel = function(event) {
 };
 
 var updateDownloadPdfCustomSelectionIndicators = function (event) {
-    if (event.data.leftPageNumber == null || event.data.rightPageNumber == null || event.data.leftPageNumber === 0) {
+    if (event.data.leftPageNumber == null || event.data.leftPageNumber === 0) {
```

Comment:
If the first page spread is currently being displayed and there is no intro page, then this is true (specifically `event.data.leftPageNumber` is 0), so the left checkbox is hidden...